RecoPixelVertexing/PixelLowPtUtilities/python/ClusterShapeHitFilterESProducer_cfi.py

```diff
@@ -11,6 +11,5 @@
 )
 from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
 phase2_tracker.toModify(ClusterShapeHitFilterESProducer,
-    PixelShapeFile = 'RecoPixelVertexing/PixelLowPtUtilities/data/pixelShape_Phase1Tk.par',
-    doPixelShapeCut = cms.bool(False)
+    PixelShapeFile = 'RecoPixelVertexing/PixelLowPtUtilities/data/pixelShape_Phase2Tk.par',
```

Comment:
You are right. 
I am not sure how to menage this problem in the different geometries.
@kpedro88 @atricomi @boudoul do you have any suggestion?